charts/pulsar/templates/bookkeeper/bookkeeper-autorecovery-statefulset.yaml

```diff
@@ -145,7 +145,7 @@ spec:
         - >
           bin/apply-config-from-env.py conf/bookkeeper.conf;
           {{- include "pulsar.autorecovery.zookeeper.tls.settings" . | nindent 10 }}
-          bin/bookkeeper autorecovery
+          OPTS="${OPTS} -Dlog4j2.formatMsgNoLookups=true" bin/bookkeeper autorecovery
```

Comment:
@MarvinCai Can you use `PULSAR_EXTRA_OPTS` to pass the flag in? You can add it into configmap instead of modifying the commandline.

Reply:
@sijie updated, PTAL